suite-common/wallet-core/src/blockchain/blockchainReducer.ts

```diff
@@ -166,3 +167,18 @@ export const selectBlockchainState = (state: BlockchainRootState) => state.walle
 export const selectNetworkBlockchainInfo =
     (networkSymbol: NetworkSymbol) => (state: BlockchainRootState) =>
         state.wallet.blockchain[networkSymbol];
+
+export const selectBlockchainHeightBySymbol = memoizeWithArgs(
+    (state: BlockchainRootState, symbol: NetworkSymbol) => {
+        const blockchain = selectNetworkBlockchainInfo(symbol)(state);
+        return blockchain.blockHeight;
+    },
+);
+
+export const selectBlockchainExplorerBySymbol = memoizeWithArgs(
+    (state: BlockchainRootState, symbol?: NetworkSymbol) => {
```

Comment:
Why is `symbol` optional? It doesn't make sense IMHO

Reply:
Because if we're getting transaction from selector, the symbol is not guaranteed, so it's optional